pkgs/applications/misc/writefreely/default.nix

```diff
@@ -13,10 +13,19 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-CBPvtc3K9hr1oEmC+yUe3kPSWx20k6eMRqoxsf3NfCE=";
 
-  nativeBuildInputs = [ go-bindata ];
+  nativeBuildInputs = [ go-bindata lessc ];
 
   preBuild = ''
     make assets
+    export NODE_PATH=${nodePackages.less-plugin-clean-css}/lib/node_modules
```

Comment:
Would that be possible to extract the assets into another drv called `writefreely-assets` and `writefreely` would depend on it?

Reply:
(even if it has some trivial phases.)